creatorkit/Project 1.md

```diff
@@ -0,0 +1,40 @@
+![Creator Logo](../creatorlogo.png)
+
+# Project 1 - IoT Button
+
+## Introduction
+
+This project will allow you to press a button on one of your Clicker boards and see an LED toggle on the other Clicker board. Ci40 acts as the hub for the 6LoWPAN communication. You can also view the inter-device messages via the FlowM2M website - demonstrating connectivity with the cloud.
+
+This project requires the Ci40 and 2 Clicker boards from Creator Kit. It does not require any of the 3 Click expansion boards.
+
+## Steps
+
+To complete these steps, refer to the [Toolbox](Toolbox.md) for detailed descriptions of each action. You will need a [FlowM2M account](Toolbox.md#creating-a-flowm2m-account) too.
+
+* Get the code: [Pre-built Release](http://github.com/creatorkit) or [Source Code](http://github.com/creatorkit)
```

Comment:
The links aren't going to be correct until we finish. Instructions for getting the source code are all in the Openwrt.md and will be referenced from here.